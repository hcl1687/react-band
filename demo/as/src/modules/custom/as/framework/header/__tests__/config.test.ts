import configFactory from '../config'

const config = configFactory()

describe('custom/as/framework/header/config', () => {
  it('should render correctly', () => {
    expect(config).toMatchSnapshot()
  })
})
