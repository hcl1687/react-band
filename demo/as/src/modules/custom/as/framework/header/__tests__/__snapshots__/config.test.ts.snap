// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`custom/as/framework/header/config should render correctly 1`] = `
Object {
  "decorators": Array [
    "@notice",
    "@i18n",
    "@theme",
    "@reduxStore",
  ],
  "decoratorsConfig": Object {
    "@reduxStore": Object {
      "authStore": Object {
        "actions": Array [
          "logout",
        ],
        "state": Array [
          "AUTH",
        ],
      },
      "layoutStore": Object {
        "actions": Array [
          "showLeft",
          "setLayout",
        ],
        "state": Array [
          "LEFT_STATUS",
          "LAYOUT_MODE",
        ],
      },
      "teacherStore": Object {
        "state": Array [
          "teacher",
        ],
      },
    },
  },
  "name": "header",
}
`;
